File: java/0029_digit_count_check.java
import java.util.Scanner;
public class number_teller
{
    public static void main(String args[])
    {
        System.out.println("This program checks whether a number has a, two or more digits."); 
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter a number:");
        double a = scanner.nextDouble();
        if(a/10==0)
        System.out.println("It is a single digit number.");
        else if (a/100==0)
        System.out.println("It is a double digit number.");
        else
        System.out.println("It has more digits.");
        scanner.close();
    }
}

File: python/0029_digit_count_check.py
num = float(input("Enter a number: "))
if num.is_integer():
    if 0 <= abs(num) <= 9:
        print(f"{num} is a 1-digit number.")
    elif 10 <= abs(num) <= 99:
        print(f"{num} is a 2-digit number.")
    elif 100 <= abs(num) <= 999:
        print(f"{num} is a 3-digit number.")
    else:
        print(f"{num} has more than 4 digits.")
else:
    print("ERROR! THIS PROGRAM ONLY SUPPROTS INTEGERS.")
